.npmrc

```
@@ -1,2 +1,3 @@
 # Ensure we always use public packages
 registry=https://registry.npmjs.org/
+@sap:registry=https://registry.npmjs.org/
```

```suggestion
@sap:registry=https://registry.npmjs.org/

```

trailing newlines. :)